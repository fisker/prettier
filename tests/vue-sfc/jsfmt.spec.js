run_spec(__dirname, ["vue-sfc"]);
