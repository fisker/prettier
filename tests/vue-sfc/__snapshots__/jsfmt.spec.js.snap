// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example.vue format 1`] = `
====================================options=====================================
parsers: ["vue-sfc"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<!-- https://vue-loader.vuejs.org/spec.html#intro -->

<template><div class="example">{{ msg
}}</div></template>

<script>
export default {
  data () {return {msg: 'Hello world!',
    }
  }
}
</script>

<style lang="scss" scoped>
.example {color: red}
</style>

<custom1>
  This could be e.g. documentation for the component.
</custom1>

=====================================output=====================================
<template>
  <div class="example">{{ msg }}</div>
</template>

<script>
export default {
  data() {
    return { msg: "Hello world!" };
  },
};
</script>

<style lang="scss" scoped>
.example {
  color: red;
}
</style>

<custom1>
  This could be e.g. documentation for the component.
</custom1>
================================================================================
`;
